components/event_dispatcher/introduction.rst

```diff
@@ -496,7 +496,7 @@ Eventi e ascolatori consapevoli del distributore
 ``EventDispatcher`` inietta sempre l'evento distribuito, il nome dell'evento
 e un riferimento a sé stesso agli ascoltatori. Questo può portare ad applicazioni
 avanzate per ``EventDispatcher``, incluse la possibilità per gli ascoltatori di distribuire altri eventi,
-il concatenamento degli eventi o anche il caricamento pigro di più ascoltatori
+il concatenamento degli eventi o anche il caricamento pigro ("lazy") di più ascoltatori
 nell'oggetto distributore. Ecco degli esempi:
```

Comment:
Scusami, ma se dovessimo fare così per ogni termine tradotto, dovremmo ripetere il termine in inglese in ogni punto in cui viene citato la prima volta. Siccome sarebbe bello mantenere una certa coerenza, o lo facciamo ovunque o non lo facciamo da nessuna parte. E siccome probabilmente farlo ovunque non è pensabile, sarebbe bene non farlo da nessuna parte.